docs/javascript/v2/guides/javascript-quickstart.mdx

```diff
@@ -3,154 +3,306 @@ title: JavaScript Quickstart Guide
+
+You can initialize a project using `npm init` or `yarn init`. While initializing the project you can enter the details for your project as you wish.
+
+### Install ParcelJS & 100ms SDK
+
+Then, install ParcelJS and 100ms SDK into your app using yarn or npm like below:
 
 ```bash section=InstallingTheDependencies sectionIndex=1
-npm install --save @100mslive/hms-video-store@latest
+yarn add --dev parcel @100mslive/hms-video-store
 ```
 
-### Initializing the SDK
+or
 
-Let us start with initializing the libraries. We need an instance of `HMSStore` and `HMSActions` to get started. Copy the code snippet below and paste it into your project.
+```bash section=InstallingTheDependencies sectionIndex=2
+npm install --save-dev parcel @100mslive/hms-video-store
+```
+
+### Project setup
 
-```jsx:script.js section=InitializingTheSDK sectionIndex=1
-import { HMSReactiveStore } from '@100mslive/hms-video-store';
+Now that ParcelJS is installed, let's create basic source files for our video-conferencing application. Let’s create the files and folders in the below manner:
 
-const hms = new HMSReactiveStore();
-const hmsStore = hms.getStore();
-const hmsActions = hms.getHMSActions();
+```
+ParcelJS-test
+  - index.html
+  - src
+    - index.js
+    - styles.css
 ```
 
-`hmsStore` will hold the complete state of the application such as details of all the participants. We can also visualize this state at any time using the
-[devtools](https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd?hl=en) extension.
+## Add basic functionalities
 
-`hmsActions` will help us perform actions such as joining the room, mute our audio and send messages.
+In this section, we will walk you through instructions to add basic functionalities for a video conferencing application so that you can test the app locally to see how two or more peers can join this app to communicate with each other.
 
 ### Concepts
 
--   `Room`: When we join a conference call, the participants are said to be in a video call `room`.
+Let's look at some basic concepts to understand how a video conferencing app works.
+
+-   `Room`: A `room` is a virtual space where one or more peers communicate with each other. It is a central entity that provides real-time communication capabilities for any video conferencing or live streaming session. To allow users to join a 100ms video conferencing or a live streaming session inside your app, you first need to create a room.
 -   `Peer`: A participant in the video call. You are the `local peer` while others are `remote peers`.
 -   `Track`: Media. There are two types of track a peer can have - audio and video.
 
-## Joining a room
+### Initialize the SDK
+
+Let us start with initializing the libraries. We need an instance of [HMSStore](/api-reference/javascript/v2/interfaces/HMSStore) and [HMSActions](/api-reference/javascript/v2/interfaces/HMSActions) to get started. Copy the code snippet below and paste it into your project.
```

Comment:
From we need an instance, seems redundant since we do the same in the code.

Reply:
simplified it.